x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/rank/textsimilarity/TextSimilarityRankRetrieverBuilder.java

```diff
@@ -39,6 +40,8 @@ public class TextSimilarityRankRetrieverBuilder extends CompoundRetrieverBuilder
         "text_similarity_reranker_alias_handling_fix"
     );
 
+    public static final String DEFAULT_RERANK_V1_INFERENCE_ID = DEFAULT_RERANK_ID;
```

Comment:
Since `DEFAULT_RERANK_ID` is already `.rerank-v1-elasticsearch` is there a need to copy this into our own variable? Presumably, reranking isn't as sensitive as ELSER since changing it doesn't require reindexing and we should just depend on the defaults defined by our internal service?